mlrun/projects/project.py

```diff
@@ -3480,6 +3542,7 @@ def list_model_endpoints(
         labels: Optional[list[str]] = None,
         start: str = "now-1h",
         end: str = "now",
+        metrics: Optional[list[str]] = None,
```

Comment:
This can be work only with v3io tsdb, we are changing `list_model_endpoints` in #6796.

Why did you add this here? Note that it is not related to the application metrics.

Reply:
removed